lib/Swagger/src/Discussion/Models/VotingResponse.php

```diff
@@ -0,0 +1,289 @@
+     */
+    public function listInvalidProperties()
+    {
+        $invalid_properties = array();
+        return $invalid_properties;
+    }
+
+    /**
+     * validate all the properties in the model
+     * return true if all passed
+     *
+     * @return bool True if all properteis are valid
+     */
+    public function valid()
+    {
+        return true;
+    }
+
+
+    /**
+     * Gets requester_id
+     * @return string
+     */
+    public function getRequesterId()
+    {
+        return $this->container['requester_id'];
+    }
+
+    /**
+     * Sets requester_id
+     * @param string $requester_id
+     * @return $this
+     */
+    public function setRequesterId($requester_id)
+    {
+        $this->container['requester_id'] = $requester_id;
+
+        return $this;
+    }
+
+    /**
+     * Gets upvote_count
+     * @return int
+     */
+    public function getUpvoteCount()
+    {
+        return $this->container['upvote_count'];
+    }
+
+    /**
+     * Sets upvote_count
+     * @param int $upvote_count
+     * @return $this
+     */
+    public function setUpvoteCount($upvote_count)
+    {
+        $this->container['upvote_count'] = $upvote_count;
+
+        return $this;
+    }
+
+    /**
+     * Gets _links
+     * @return map[string,\Swagger\Client\Discussion\Models\NewHalLink]
+     */
+    public function getLinks()
+    {
+        return $this->container['_links'];
+    }
+
+    /**
+     * Sets _links
+     * @param map[string,\Swagger\Client\Discussion\Models\NewHalLink] $_links
+     * @return $this
+     */
+    public function setLinks($_links)
+    {
+        $this->container['_links'] = $_links;
+
+        return $this;
+    }
+    /**
+     * Returns true if offset exists. False otherwise.
+     * @param  integer $offset Offset
+     * @return boolean
+     */
+    public function offsetExists($offset)
+    {
+        return isset($this->container[$offset]);
+    }
+
+    /**
+     * Gets offset.
+     * @param  integer $offset Offset
+     * @return mixed
+     */
+    public function offsetGet($offset)
+    {
+        return isset($this->container[$offset]) ? $this->container[$offset] : null;
+    }
+
+    /**
+     * Sets value based on offset.
+     * @param  integer $offset Offset
+     * @param  mixed   $value  Value to be set
+     * @return void
+     */
+    public function offsetSet($offset, $value)
+    {
+        if (is_null($offset)) {
+            $this->container[] = $value;
+        } else {
+            $this->container[$offset] = $value;
+        }
+    }
+
+    /**
+     * Unsets offset.
+     * @param  integer $offset Offset
+     * @return void
+     */
+    public function offsetUnset($offset)
+    {
+        unset($this->container[$offset]);
+    }
+
+    /**
+     * Gets the string presentation of the object
+     * @return string
+     */
+    public function __toString()
+    {
+        if (defined('JSON_PRETTY_PRINT')) { // use JSON pretty print
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)